src/mini_file_server/core/handler.clj

```diff
@@ -32,10 +33,12 @@
       response
       (content-type "application/x-compressed")))
 
-(defroutes app-routes
-  (route/resources "/")
-  (GET "/" [] (index/render))
-  (GET "/list.json" [] (response (fs/files)))
+(defn authenticated? [id pass]
+  (and (= id "admin")
+       (= pass "admin")
+       {:user id :passwd pass}))
```

Comment:
If I'm not mistaken, this function should return a "truthy" value when the credential is valid. Can we just remove this line because a boolean value should suffice?

Reply:
`wrap-basic-authentication` needs two parameters.

-----------

> The authenticate function is called with two parameters, the userid and password, and should return a value when the login is valid. This value is added to the request structure with the :basic-authentication key.

https://cljdoc.org/d/ring-basic-authentication/ring-basic-authentication/1.2.0/api/ring.middleware.basic-authentication#wrap-basic-authentication

Reply:
I think the return value doesn't have to be "truthy".
But, I won't use the value, so "truthy" would be enough.

ref) 
wrap-basic-authentication
(wrap-basic-authentication app authenticate & [realm denied-response])

> The authenticate function is called with two parameters, the userid and password, and should return a value when the login is valid. This value is added to the request structure with the :basic-authentication key.

https://cljdoc.org/d/ring-basic-authentication/ring-basic-authentication/1.2.0/api/ring.middleware.basic-authentication#wrap-basic-authentication

Reply:
I don't think the return value necessarily needs to be "truthy".
However, since I won't be using the value, a "truthy" return value would suffice.

ref) 
wrap-basic-authentication
(wrap-basic-authentication app authenticate & [realm denied-response])

> The authenticate function is called with two parameters, the userid and password, and should return a value when the login is valid. This value is added to the request structure with the :basic-authentication key.

https://cljdoc.org/d/ring-basic-authentication/ring-basic-authentication/1.2.0/api/ring.middleware.basic-authentication#wrap-basic-authentication